kani-driver/src/call_cbmc.rs

```diff
@@ -183,6 +183,7 @@ impl KaniSession {
 
         if self.args.checks.unwinding_on() {
             args.push("--unwinding-assertions".into());
+            args.push("--no-self-loops-to-assumptions".into());
```

Comment:
Should we add a comment here saying this must be updated once we move to CBMC 6.0?

Reply:
+1